Ago-Dic-2020/gonzalez-cardenas-jesus-antonio/Practica 2/Practica2.py

```diff
@@ -0,0 +1,95 @@
+class Vehiculo(object):
+    def __init__ (self, name, dateRelease, passengers, numOfWheels, terrain, typeVehicle):
+        self.name = name
+        self.dateRelease = dateRelease
+        self.passengers = passengers
+        self.numOfWheels = numOfWheels
+        self.terrain = terrain
+        self.typeVehicle = typeVehicle
+
+    def specs(self):
+        print(self.name+' salió en la fecha '+ self.dateRelease +', tiene el número de llantas: ' +str(self.numOfWheels) +'. Y es para el terreno tipo: ' +self.terrain)
+        print('Numero de pasajeros = ' + str(self.passengers))
+
+    def movev(self, km):
+        print(self.name +' se ha movido ' + str(km)+' unidades')
+
+    def stop(self):
+        print(self.name +' se ha detenido')
+
+class Camion(Vehiculo):
+    NumActualDePasajeros = 0
+    Cuenta = 0.000
+    cobro = 0.000
```

Comment:
No es necesario definir estas variables de clase, ya que estas son diferentes de los atributos o variables de instancia: https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3